### ISSUE TITLE
unable to install - rocksdb/db.h file not found

### ISSUE THREAD

**Issue by conoro** [dberesford/rocksdb-node#1]
When I do 

`npm i rocksdb-node --save
`
on OSX with Node 6.10.1

I get the error below. Do I need to install anything else?

Thanks!

```
> rocksdb-node@0.1.0 install /Users/conoroneill/gitwork/rocksdb-node-test/node_modules/rocksdb-node
> node-gyp rebuild

  CXX(target) Release/obj.target/rocksdb/rocksdb.o
In file included from ../rocksdb.cc:3:
../RocksDBNode.h:5:10: fatal error: 'rocksdb/db.h' file not found
#include "rocksdb/db.h"
         ^
1 error generated.
make: *** [Release/obj.target/rocksdb/rocksdb.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Darwin 16.5.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/conoroneill/gitwork/rocksdb-node-test/node_modules/rocksdb-node
gyp ERR! node -v v6.10.1
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
npm WARN rocksdb-node-test@1.0.0 No description
npm WARN rocksdb-node-test@1.0.0 No repository field.
npm ERR! Darwin 16.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "i" "rocksdb-node" "--save"
npm ERR! node v6.10.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE

```

**Comment by conoro** [comment]
Doh. Just needs

`brew install rocksdb
`

**Comment by dberesford** [comment]
Thanks Conor, I should have that in the readme!

On 4 May 2017 23:06, "conoro" <notifications@github.com> wrote:

Closed #1 <https://github.com/dberesford/rocksdb-node/issues/1>.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/dberesford/rocksdb-node/issues/1#event-1069758963>, or mute
the thread
